File: js/baekjoon/step9/bkjn_1978_1.js
const { chmod } = require("fs");

const input = require("fs")
  .readFileSync("dev/stdin")
  .toString()
  .trim()
  .split("\n");

let nums = input[1].split(" ");

for (i = 0; i < nums.length; i++) {
  let numArr = [];
  let number = Number(nums[i]);
  let index = 1;
  let primeArr = [];
  let primeNum = 0;
  while (index <= number) {
    if (number % index === 0) numArr.push(index);
    index++;
  }
  if (numArr[0] === 1 && numArr[1] === number && numArr.length === 2) {
    primeNum += 1;
    console.log(primeNum);
  }
}

File: python/baekjoon/bkjn_1978_1.py
n = int(input())
nums = list(map(int, input().split()))
result = 0

for i in nums:
  count = 0
  if i > 1:
    for j in range(2, i):
      if i % j == 0:
        count += 1
    if count == 0:
      result += 1

print(result)
